src/lat/lattice-functions.h

```diff
@@ -74,6 +74,26 @@ bool ComputeCompactLatticeAlphas(const CompactLattice &lat,
 bool ComputeCompactLatticeBetas(const CompactLattice &lat,
                                 vector<double> *beta);
 
+
+static inline double LogAddOrMax(bool viterbi, double a, double b) {
```

Comment:
Its a small function that can be inlined. If it is moved to .cc file, it
cannot be inlined.

On Sun, Mar 6, 2016 at 8:19 PM Daniel Povey <notifications@github.com>
wrote:

> In src/lat/lattice-functions.h
> <https://github.com/kaldi-asr/kaldi/pull/560#discussion_r55153738>:
>
> > @@ -74,6 +74,26 @@ bool ComputeCompactLatticeAlphas(const CompactLattice &lat,
> >  bool ComputeCompactLatticeBetas(const CompactLattice &lat,
> >                                  vector<double> *beta);
> >
> > +
> > +static inline double LogAddOrMax(bool viterbi, double a, double b) {
>
> do you still need this function to be in the header?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/560/files#r55153738>.
>
-- 
Vimal Manohar
PhD Student
Electrical & Computer Engineering
Johns Hopkins University